src/tree/registries/dockerV2/DockerV2TagTreeItem.ts

```diff
@@ -3,43 +3,12 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as moment from 'moment';
 import { RequestPromiseOptions } from 'request-promise-native';
-import { AzExtTreeItem } from "vscode-azureextensionui";
-import { nonNullProp } from '../../utils/nonNull';
-import { registryRequest } from '../../utils/registryRequestUtils';
-import { getThemedIconPath, IconPath } from '../IconPath';
-import { RemoteRepositoryTreeItemBase } from './RemoteRepositoryTreeItemBase';
-
-export abstract class RemoteTagTreeItemBase extends AzExtTreeItem {
-    public static contextValueSuffix: string = 'Tag';
-    public static allContextRegExp: RegExp = /Tag$/;
-    public parent: RemoteRepositoryTreeItemBase;
-    public tag: string;
-    public time: Date;
-
-    public constructor(parent: RemoteRepositoryTreeItemBase, tag: string, time: string) {
-        super(parent);
-        this.tag = tag;
-        this.time = new Date(time);
-    }
-
-    public get label(): string {
-        return this.tag;
-    }
-
-    public get fullTag(): string {
-        return this.parent.repoName + ':' + this.tag;
-    }
-
-    public get description(): string {
-        return moment(this.time).fromNow();
-    }
-
-    public get iconPath(): IconPath {
-        return getThemedIconPath('tag');
-    }
+import { nonNullProp } from '../../../utils/nonNull';
+import { registryRequest } from '../../../utils/registryRequestUtils';
+import { RemoteTagTreeItem } from '../RemoteTagTreeItem';
 
+export class DockerV2TagTreeItem extends RemoteTagTreeItem {
```

Comment:
It's not a direct renaming. One of my goals of this PR was to make it so that as much code could be shared as possible between registries. So if a registry doesn't have any unique functionality, I don't have a separate class for it. For example, Docker Hub doesn't need it's own class and can use the generic `RemoteTagTreeItem` directly. It's the same case when I did a separate PR for GitLab. It can use the generic `RemoteTagTreeItem`